future.go

```diff
@@ -172,24 +184,35 @@ func (fut *Future) SetError(err error) {
 	close(fut.done)
 }
 
-// Get waits for Future to be filled and returns Response and error.
-//
-// Response will contain deserialized result in Data field.
-// It will be []interface{}, so if you want more performance, use GetTyped method.
+// GetResponse waits for Future to be filled and returns Response and error.
 //
 // Note: Response could be equal to nil if ClientError is returned in error.
 //
 // "error" could be Error, if it is error returned by Tarantool,
 // or ClientError, if something bad happens in a client process.
-func (fut *Future) Get() (*Response, error) {
+func (fut *Future) GetResponse() (Response, error) {
 	fut.wait()
 	if fut.err != nil {
 		return fut.resp, fut.err
 	}
-	err := fut.resp.decodeBody()
+	_, err := fut.resp.Decode()
```

Comment:
Because of this change, if the user wants to get a `SQLInfo` (for example) field from the response, the code going to look like this:
```Go
	exec := tarantool.NewExecuteRequest("...")
	resp, err := conn.Do(exec).GetResponse()
	require.Nil(t, err)
	require.NotNil(t, resp)
	_, err = resp.Decode()
	require.Nil(t, err)
	exResp, ok := resp.(*tarantool.ExecuteResponse)
	require.True(t, ok, "wrong response type")
	require.Equal(t, uint64(1), exResp.SQLInfo().AffectedCount)
```

Reply:
I don't see a big problem in the code. SQLInfo() could call Decode() if ExecuteResponse.Decode() and ExecuteResponse.DecodeTyped() have been not called yet.
```Go
exec := tarantool.NewExecuteRequest("...")
resp, err := conn.Do(exec).GetResponse()
if err != nil || resp.Header().Code != OkCode {
    // An error.
}

exResp := resp.(*tarantool.ExecuteResponse)
info, err := exResp.SQLInfo()
```